src/jsutils/Path.d.ts

```diff
@@ -1,12 +1,17 @@
 export interface Path {
   prev: Path | undefined;
   key: string | number;
+  parentType: string | undefined;
```

Comment:
I'm not sure about `parentType` name since it's too tied `info` object.
I would like something that you can easily understand even if you never looked into `info`.
I actually think you touch a bigger topic here since you can't identify resolver that was triggered just from `path` so I feel like this structure would be useful for logging, debugging, and other similar purposes and in a future figure how to add it inside `GraphQLError` instead of array path.

How about `typename` (we can't use underscores since it will look like private field)?
It will also solve the confusion for JS users since context `parentType` is a `GraphQLOutputType` and here it's a string and I don't really like `parentTypeName` as an alternative.

Reply:
it will also explain why we don't have it on an array segment of the path since you will get `__typename` only on this levels in your response.

Reply:
Done